crates/storage/src/config.rs

```diff
@@ -0,0 +1,2 @@
+pub const REDB_FILE: &str = "ream.redb";
+pub const REDB_CACHE_SIZE: usize = 1_024 * 1_024 * 1_024; // 1 GiB
```

Comment:
```suggestion

/// 1 GiB
pub const REDB_CACHE_SIZE: usize = 1_024 * 1_024 * 1_024;
```

can you use a doc comment